templates/etc/httpd/conf.d/pulp_nodes.conf.erb

```diff
@@ -22,3 +22,14 @@ Alias /pulp/nodes/https/repos /var/www/pulp/nodes/https/repos
   SSLOptions +FakeBasicAuth
   SSLRequire %{SSL_CLIENT_S_DN_O} eq "PULP" and %{SSL_CLIENT_S_DN_OU} eq "NODES"
 </Directory>
+
+Alias /pulp/nodes/content /var/www/pulp/nodes/content
+
+<Directory /var/www/pulp/nodes/content >
+  Options FollowSymLinks Indexes
+  SSLRequireSSL
+  SSLVerifyClient require
```

Comment:
Looks like this should follow https://github.com/Katello/puppet-pulp/pull/116/files#diff-56c104ac5f76e0676fddcfb2da13dd62R20